docker-compose.override.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.4'
+
+services:
+  http:
+    # See instructions if using a custom domain for dev.
+    # extra_hosts:
+    #   - "osu-web.test:192.168.65.x"
+    ports:
+      - "${HTTP_PORT:-4001}:4001"
+    volumes:
+      # override packaged /app for dev
+      - .:/app
+      # use node_modules from the container, not the host.
+      - node_modules:/app/node_modules
```

Comment:
Note this rely on yarnpkg's behavior to fetch packages before running (as this volume is initially empty), which npm doesn't.

I am not sure why we would not want to use node_modules from the host anyway?

Reply:
These node_modules are not compatible across platforms; this is the de facto way for handling node_modules in dev for these scenarios.